dotcom-rendering/src/components/RichLinkComponent.importable.tsx

```diff
@@ -79,68 +78,72 @@ export const RichLinkComponent = ({
 	richLinkIndex,
 	format,
 }: Props) => {
+	/** Render a default (basic) rich link on the server */
+	const fallbackData = {
+		headline: element.text,
+		url: element.url,
+		thumbnailUrl: '',
+		imageAsset: {
+			fields: {
+				altText: '',
+				width: '',
+				height: '',
+				source: '',
+				displayCredit: '',
+				photographer: '',
+				isMaster: '',
+				credit: '',
+				mediaId: '',
+			},
+			index: -1,
+			mediaType: '',
+			url: '',
+		},
+		contentType: 'article',
+		format: {
+			design: 'ArticleDesign',
+			display: 'StandardDisplay',
+			// We default to SpecialReport here purely because the greys of this theme
+			// look better as the defaults
+			theme: 'SpecialReportTheme',
+		},
+		cardStyle: 'news',
+		tags: [],
+		sponsorName: '',
+	} satisfies FERichLinkType;
+
 	const url = buildUrl(element, ajaxUrl);
-	const { data, error } = useApi<FERichLinkType>(url);
+	const { data, error } = useApi<FERichLinkType>(url, { fallbackData });
 
 	if (error) {
 		// Send the error to Sentry
 		window.guardian.modules.sentry.reportError(error, 'rich-link');
 	}
 
-	if (data?.imageAsset) {
-		const richLinkImageData: RichLinkImageData = {
-			thumbnailUrl: data.thumbnailUrl,
-			altText: data.imageAsset.fields.altText,
-			width: data.imageAsset.fields.width,
-			height: data.imageAsset.fields.height,
-		};
+	if (!data) return null;
 
-		return (
-			// This is the enhanced rich link rendered on the client
-			<RichLink
-				richLinkIndex={richLinkIndex}
-				cardStyle={data.cardStyle}
-				imageData={richLinkImageData}
-				headlineText={data.headline}
-				contentType={data.contentType}
-				url={data.url}
-				starRating={data.starRating}
-				linkFormat={decideFormat(data.format)}
-				format={format}
-				tags={data.tags}
-				sponsorName={data.sponsorName}
-				contributorImage={data.contributorImage}
-			/>
-		);
-	}
-
-	const defaultFormat = {
-		display: ArticleDisplay.Standard,
-		design: ArticleDesign.Standard,
-		// We default to SpecialReport here purely because the greys of this theme
-		// look better as the defaults
-		theme: ArticleSpecial.SpecialReport,
+	const richLinkImageData: RichLinkImageData = {
+		thumbnailUrl: data.thumbnailUrl,
+		altText: (data.imageAsset ?? fallbackData.imageAsset).fields.altText,
+		width: (data.imageAsset ?? fallbackData.imageAsset).fields.width,
+		height: (data.imageAsset ?? fallbackData.imageAsset).fields.height,
```

Comment:
It will not once we receive real data, as there are no guarantees that the image asset will be present as it’s [optional on `FERichLinkType`](https://github.com/guardian/dotcom-rendering/blob/d3ffab7ad6571a29dc320365735e070f79121360/dotcom-rendering/src/components/RichLinkComponent.importable.tsx#L25), and [in the `frontend` data model](https://github.com/guardian/frontend/blob/8256fe148f9abb5842e30339ac64dfa43d758226/onward/app/models/dotcomrendering/DotcomponentsOnwardsModels.scala#L21).